mix.exs

```diff
@@ -83,7 +83,7 @@ defmodule Membrane.VideoCompositor.Mixfile do
         ["lib", "mix.exs", "README*", "LICENSE*", ".formatter.exs"] ++
           Enum.map(
             ["src", ".cargo/config", "Cargo.toml", "Cargo.lock"],
-            &"native/membrane_videocompositor/#{&1}"
+            &"native/membrane_video_compositor/#{&1}"
```

Comment:
I actually am not sure what this field is for

Reply:
I believe it's used to describe which files should be put in a package

Reply:
I don't really know whether to put it here, since it depends on how we want to distribute `membrane_video_compositor_common`. I guess I will put it here for now and we can think about this in a more opportune time.